lib/devise/controllers/sign_in_out.rb

```diff
@@ -106,10 +106,12 @@ def sign_out_all_scopes(lock = true)
       private
 
       def expire_data_after_sign_in!
+        # TODO: remove once Rails 5.2+ and forward are only supported.
```

Comment:
I personally would be fine with having the answer to why the `TODO` was added only in the commit message. 

But I am not sure and would love to get someone else's opinion

Reply:
Having the reasoning in the commit message only is fine in this case, we can trace if down if needed from git. Thanks.